GenderPayGap.WebUI/Views/Viewing/Finder/Parts/Facets/Keywords.cshtml

```diff
@@ -28,22 +45,30 @@
         <div class="gov-uk-c-searchbar">
             <div class="gov-uk-l-searchbar__table">
                 <div class="gov-uk-l-searchbar__cell">
-                    <input id="FinderKeywords"
-                           value="@(Model.search)"
-                           aria-expanded="false"
-                           class="gov-uk-c-searchbar__input"
-                           name="search"
-                           type="text"
-                           autocomplete="off"
-                           spellcheck="false"
-                           maxlength="100"/>
+                    <label id="SearchInputLabel"for="FinderKeywords" class="visuallyhidden">
+                        @(!string.IsNullOrEmpty(Model.t) && Model.t == "1" 
+                            ? "Search by employer name" : "Search by employer type or SIC code")
+                    </label>
+                    <input class="gov-uk-c-searchbar__input" type="text" autocomplete="off" name="search" id="FinderKeywords">
                 </div>
                 <div class="gov-uk-l-searchbar__cell">
-                    <button class="gov-uk-c-searchbar__submit gov-uk-c-searchbar__submit--blue" id="NextStep" type="submit">Search</button>
+                    <button class="gov-uk-c-searchbar__submit gov-uk-c-searchbar__submit--blue" id="NextStep" type="submit">
+                        Search
+                    </button>
                 </div>
-
             </div>
         </div>
     </fieldset>
-
-</div>
\ No newline at end of file
+</div>
+searchby

```

Comment:
Is this line deliberate?

Reply:
Definitely not, I've removed that now. Thanks